assert/assert-filter/src/main/java/com/mastercard/test/flow/assrt/filter/Filter.java

```diff
@@ -387,12 +410,14 @@ private boolean matches( Flow flow ) {
 		if( !includeTags.isEmpty() ) {
 			Set<String> includeIntersection = new HashSet<>( flow.meta().tags() );
 			includeIntersection.retainAll( includeTags );
-			included &= !includeIntersection.isEmpty();
+			// the flow bears *all* of the included tags
+			included &= includeIntersection.size() == includeTags.size();
```

Comment:
So it turns out that code coverage is _not_ the same thing as correctness. Who knew!?!?